pkgs/applications/terminal-emulators/germinal/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, appstream-glib
+, dbus
+, pcre2
+, tmux
+, vte
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "germinal";
+  version = "26";
+
+  src = fetchFromGitHub {
+    owner = "Keruspe";
+    repo = "Germinal";
+    rev = "v${version}";
+    sha256 = "sha256-HUi+skF4bJj5CY2cNTOC4tl7jhvpXYKqBx2rqKzjlo0=";
```

Comment:
IIRC we are trying to not use the `sha-…` type of hashes in nixpkgs. At least not when assigning to the `sha256` attribute.

Reply:
Is this true? I've mostly just given up and started putting any type of hash (sha256: prefixed base32, non-prefixed base32, hex, these SRI hashes...) in any of sha256|hash depending on what time of day it is and how hungry I am...

Reply:
@andir I am using the nix command, and it throws out hashes in SRI format now.